.github/workflows/ci.yaml

```diff
@@ -44,17 +50,14 @@ jobs:
       #    chmod 0755 run.sh
       #    ./run.sh bootstrap
 
-      - name: Container
-        run: docker pull ${{ matrix.cntr }}
-
       - name: SessionInfo
-        run: docker run --rm -i -v ${PWD}:/mnt -w /mnt ${{ matrix.cntr }} ${{ matrix.r }} -q -e 'sessionInfo()'
+        run: ${{ matrix.r }} -q -e 'sessionInfo()'
 
       - name: Build
-        run: docker run --rm -i -v ${PWD}:/mnt -w /mnt ${{ matrix.cntr }} ${{matrix.r }} CMD build --no-build-vignettes --no-manual .
+        run: ${{matrix.r }} CMD build --no-build-vignettes --no-manual .
 
       - name: Check
-        run: docker run --rm -i -v ${PWD}:/mnt -w /mnt -e CI=true ${{ matrix.cntr }} ${{ matrix.r }} CMD check --no-vignettes --no-manual Rcpp_*.tar.gz
+        run: ${{ matrix.r }} CMD check --no-vignettes --no-manual Rcpp_*.tar.gz
```

Comment:
Note that `CI=true` was dropped.

Reply:
Fixed -- and that's why we have more than two eyes on these.  Thanks!